docs/content/jargon-dictionary.md

```diff
@@ -73,6 +91,11 @@ The NATbox is an optional Wakame-vdc node that provides one to one [network addr
 Refers to a server that a Wakame-vdc process runs on. Can be physical or a VM but not an [instance](#instance).
 
 ## O
+
+### OpenVNet
+
+An full virtual networking implementation using OpenFlow. This project is completely separate from Wakame-vdc but Wakame-vdc can use it to implement virtual networking. This integration is still experimental at this time. [http://openvnet.org](http://openvnet.org)
```

Comment:
'A full' ? :-) I'd remove the 'experimental' bit, no need for that, it's openvnet related info (should be located ith openvnet docs) and it'll change at some point.